index.html

```diff
@@ -1,16 +1,26 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <link rel='stylesheet' id='cookie-consent-style-css'  href='https://www.munroandpartners.com/wp-content/plugins/uk-cookie-consent/assets/css/style.css?ver=4.9.6' type='text/css' media='all' />
```

Comment:
@kevin-chilcott-sift  Can you remove  all the css references to munroandpartners. This repo is public so lets avoid having these in there.

Reply:
I have just removed all references to munro and partners.